components/Typography/docs/dynamic-type.md

```diff
@@ -0,0 +1,33 @@
+### Dynamic Type
+
+#### Overview
+Typography library provides APIs that allow users to support Dynamic Type feature in Material Components. For those components with Dynamic Type feature supported, text size changes based on the Dynamic Type setting provided by the user.
```

Comment:
Consider replacing "For those components with Dynamic Type feature supported, text size changes based on the Dynamic Type setting provided by the user." with a "Lean more about Dynamic Type" link to Apple's marketing material on Dynamic Type (not the engineering docs, because those may cause confusion for the reader if they assume we support Dynamic Type's APIs 1 to 1).

Reply:
I tried to find it as reference. And surprisingly, https://www.apple.com/accessibility/ is closest one I could find and was not well documented. Also, I agree with we shouldn't reference to Apple's engineering doc here to avoid confusion.

Reply:
I expect the audience who look at this document are Apple developers so that Dynamic Type is something they are familiar with.

Reply:
Bummer. Let's drop the "For those components with Dynamic Type feature supported, text size changes based on the Dynamic Type setting provided by the user." part then if we are assuming that the user is aware of how Dynamic Type works.

Reply:
Addressed in 32e2339180fe1be49a20bb7e3931d105f363a0dd.